src/main/java/com/sofa/linkiving/domain/link/dto/OgTagDto.java

```diff
@@ -0,0 +1,15 @@
+package com.sofa.linkiving.domain.link.dto;
+
+import lombok.Builder;
+
+@Builder
+public record OgTagDto(
+	String title,
+	String description,
+	String image,
+	String url
+) {
+	public static OgTagDto empty() {
+		return OgTagDto.builder().build();
+	}
```

Comment:
OgTagDto는 **Record(불변 객체)**이므로, 비어있는 객체의 상태는 언제나 동일합니다. 따라서 매번 메서드로 새로운 객체를 생성(new)하기보다, static final 상수로 선언해서 재사용하는 것이 메모리 효율 측면에서 더 좋을 것 같습니다.

또한, 현재 builder().build()로 생성하면 필드값이 모두 null이 되어 사용처에서 **NPE(NullPointerException)**가 발생할 위험이 있습니다. null 대신 **빈 문자열("")**로 초기화해두면 별도의 null 체크 없이 더 안전하게 사용할 수 있을 것 같습니다.

```suggestion
public static final OgTagDto EMPTY = new OgTagDto("", "", "", "");
```